staging/src/k8s.io/component-base/config/v1alpha1/defaults.go

```diff
@@ -44,7 +44,7 @@ func RecommendedDefaultLeaderElectionConfiguration(obj *LeaderElectionConfigurat
 		obj.RetryPeriod = metav1.Duration{Duration: 2 * time.Second}
 	}
 	if obj.ResourceLock == "" {
-		// TODO: Migrate to LeaseLock.
+		// TODO(#80289): Migrate to LeaseLock when graduating to v1beta1.
```

Comment:
@liggitt - who owns that part now?
I would be happy to promote it to v1beta1 in 1.20 to clean this up, but wanted to check with the owner if there aren't any known blockers for doing that (or other things we want to fix during this migration).

Reply:
I'm not sure... cc @mtaufen?

Reply:
I don't think it's a particularly active file, but it's a core dependency so it would be good to stabilize the version. I'm not sure what the blockers would be off the top of my head. I'm happy to help if you want to work on moving it forward.

Reply:
OK - I will try to get back to in in the next couple weeks.

Reply:
Ok, no rush as I'll be OOO for the next couple weeks anyway.